docs/tutorials/projects/tipsy.md

```diff
@@ -839,10 +757,10 @@ To make Tipsy even more advanced, you can try to:
 
 - Add more ultrasonic sensors so it doesn’t hit objects at different heights, you can also attach them to a moving gantry along the side rails
 - Add a depth camera to detect obstacles and how close they are to Tipsy
-- Add an imu to see if Tipsy is tipping backward
+- Add an IMU to see if Tipsy is tipping backward
 - Add a lidar
 
-You can also design another robot for collecting the empty beer cans, or a bartender robot with pumps that can mix some drinks.
-Till then, sit back, relax, and let Tipsy handle the beer-carrying duties for you!
+You can also design another robot for collecting the empty drink cans, or a bartender robot with pumps that can mix some drinks.
+Until then, sit back, relax, and let Tipsy handle the drink-carrying duties for you!
```

Comment:
I updated "beer" to "drink" for consistency since we're calling it a "Drink Carrying Robot" in the title.

Reply:
thank you